src/features/content-answers/Answer.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import ContentAnswersGridCard from './ContentAnswersGridCard';
+import LoadingElement from './LoadingElement';
+
+import './Answer.scss';
+
+type Props = {
+    answer?: string;
+    isLoading: boolean;
+};
+
+const Answer = ({ answer, isLoading }: Props) => {
+    return (
+        <div className="bdl-Answer">
+            {answer && <ContentAnswersGridCard>{answer}</ContentAnswersGridCard>}
+            {!answer && isLoading && <LoadingElement />}
```

Comment:
Is it possible for the answer to come back empty and isLoading be false? If so should the decision whether to show the LoadingElement by controlled solely by the isLoading prop?

Reply:
There should always be a response but in the event of an error but not a network error, the response has a prop that can say `error` as the `completion_reason`. That error logic will be handle in a different PR prior to entering this component. Network error will fall into the same logic but again a different PR

Reply:
Ok so what will a user see if answer is empty but the isLoading value is false for some reason?

Reply:
Will be made moot in a subsequent PR as per convo with @greg-in-a-box